src/streams/CreateStreamScreen.js

```diff
@@ -20,13 +22,19 @@ type Props = $ReadOnly<{|
 export default function CreateStreamScreen(props: Props): Node {
   const auth = useSelector(getAuth);
   const ownEmail = useSelector(getOwnEmail);
+  const getStreams = useSelector(getStreamsByName);
+  const allStreams = Array.from(getStreams.keys());
 
   const handleComplete = useCallback(
     (name: string, description: string, isPrivate: boolean) => {
-      api.createStream(auth, name, description, [ownEmail], isPrivate);
-      NavigationService.dispatch(navigateBack());
+      if (allStreams.includes(name)) {
+        showErrorAlert('A Stream with this name already exists');
```

Comment:
Lowercase "stream", and put a period at the end: "A stream with this name already exists."

Then, please add an entry to static/translations/messages_en.json, as mentioned in our translations doc (docs/howto/translations.md).